index.html

```diff
@@ -310,6 +319,224 @@ <h2>Contact Me</h2>
       
   <hr>
     </section> 
-  
+    <section class="works-desktop">
+      <div class="card-work1">
+        <div class="card-image">
+          <img src="./images/d1.jpg" alt="card-work">
+        </div>
+        <h2 class="h2">Tonic</h2>
+      <div class="left-block">
+          <div class="frame1">
+            <p>CONOPY</p>
+            <img src="./images/Counter.jpg" alt="">
+            <p class="p1">Back End Dev</p>
+            <img src="./images/Counter.jpg" alt="">
+            <p class="p1">2020</p>
+          </div>
+          
+            <p class="primary1">
+              A daily selection of privately<br />personalized reads; 
+            no accounts or<br /> sign-ups required.
+          </p>
+          <ul>
+            <li><a href="#">html</a></li>
+            <li><a href="#">css</a></li>
+            <li><a href="#">javascript</a></li>
+          </ul>
+          <div class="action">
+            <a href="#"><p>See Project</p></a>
+          </div>
```

Comment:
- [ ] This `See Project` button should take you to the Live Demo of the project or to the GitHub repository. So please implement it